robottelo/ui/hosts.py

```diff
@@ -77,6 +93,21 @@ def _configure_interface_parameters(self, parameters_list):
                 'host',
                 (parameter_name.lower()).replace(' ', '_')
             ))]
+            # send_keys() can't send left parenthesis (see
+            # SeleniumHQ/selenium#674), which is a part of network type name
+            # (e.g. 'Physical (Bridge)')
+            if parameter_name == 'Network type' and ' (' in parameter_value:
```

Comment:
Good point. Maybe it's even worth it to update `Base.select()` to automatically apply this workaround whenever left parenthesis is found in 'new style' drop downs. I'll submit new issue and think of it - such core change will require additional testing and should probably be done in a separate PR, that will be framework improvement, not fix for hosts.